manifests/t/Tutanota/Tutanota/3.105.4/Tutanota.Tutanota.locale.en-US.yaml

```diff
@@ -32,9 +32,11 @@ Tags:
 - secure
 # Agreements:
 # ReleaseNotes:
-# ReleaseNotesUrl:
-# PurchaseUrl:
+ReleaseNotesUrl:
```

Comment:
```suggestion
# ReleaseNotesUrl:
```
I think this might have been a mistake: either needs commenting or a value added to it

Reply:
oopsies, thanks for catching this